macos/setup_sound.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
 set -xeo pipefail
+brew fetch --retry --cask background-music
```

Comment:
@mscheltienne I saw [a download error](https://github.com/LABSN/sound-ci-helpers/actions/runs/14737302207/job/41366543983) so I added this. Should help hopefully. Is that the error you were seeing or something else?

Reply:
Something else, and it's not present anymore. Thanks :)